canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -167,6 +172,26 @@ class RadiosetControl extends React.Component {
 		let wasChecked = false;
 		const valueSet = this.props.controlOpts;
 		for (var i = 0; i < valueSet.values.length; i++) {
+			const tooltipId = uuid4() + "-tooltip-" + this.props.control.name;
+			if (this.props.control.valueDescs && !(this.props.state === STATES.DISABLED || this.props.state === STATES.HIDDEN) && !this.props.tableControl) {
```

Comment:
We should move up `this.props.state === STATES.DISABLED || itemDisabled` to above where `disabled` is set and use that in checks.  

I'm not sure we need `this.props.state === STATES.HIDDEN` in this check since the entire control would be hidden.  Maybe this is needed because of the placement of the tooltip below.